api/syncStatusApi.ts

```diff
@@ -171,75 +170,4 @@ export class SyncStatusApi {
             });
         });
     }
-    /**
-     * Force re-sync of all models. This is only available for organizations on Merge\'s Grow and Expand plans.
-     * @param xAccountToken Token identifying the end user.
-     */
-    public async syncStatusResyncCreate (xAccountToken: string, options: {headers: {[name: string]: string}} = {headers: {}}) : Promise<{ response: http.IncomingMessage; body: SyncStatus;  }> {
```

Comment:
should we confirm that no one is still using this before removing?

Reply:
Ah i was originally confused about this too since i was sure i did not manually remove it: it has been renamed to forceResyncApi.ts and moved out of the syncStatusApi.ts class.